client/account-recovery/account-recovery-root/style.scss

```diff
@@ -1,9 +1,11 @@
+/** @format */
```

Comment:
apparently these comments get shipped into production css which is not what we want. may revisit with another approach

Reply:
Not ideal but let's finish one pass before we touch everything again 🙈